### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: src/Files.Uwp/Strings/de-DE/Resources.resw [hunk 1]
```diff
@@ -2791,4 +2791,7 @@ Wir benutzen das App Center, um einen Überblick über die Nutzung von Files zu
   <data name="BaseLayoutContextFlyoutAdaptive.KeyboardAcceleratorTextOverride" xml:space="preserve">
     <value>Ctrl+Shift+7</value>
   </data>
+  <data name="ShowAlternateStreams" xml:space="preserve">
+    <value>Show Alternate Data Streams</value>
```
Comment (chingucoding) [inline src/Files.Uwp/Strings/de-DE/Resources.resw:2795]: German translation would be:
```suggestion
    <value>Zeige alternative Daten Streams an</value>
```

Comment (yaichenbaum) [reply src/Files.Uwp/Strings/de-DE/Resources.resw:2795]: Thanks! Can you update this on Crowdin?

Comment (chingucoding) [reply src/Files.Uwp/Strings/de-DE/Resources.resw:2795]: I don't have a crowdin account I think or at least I don't know which account that could be.

Comment (yaichenbaum) [reply src/Files.Uwp/Strings/de-DE/Resources.resw:2795]: https://crowdin.com/project/files-app

Comment (chingucoding) [reply src/Files.Uwp/Strings/de-DE/Resources.resw:2795]: Updated it there and also, editing these things in Crowdin is really cool! Really glad we are using that (or at least testing it)

Comment (yaichenbaum) [reply src/Files.Uwp/Strings/de-DE/Resources.resw:2795]: Translators have been asking to use Crowdin for a while and it's easy to see why.

File: src/Files.Uwp/Strings/de-DE/Resources.resw [hunk 2]
```diff
@@ -907,10 +907,10 @@
     <value>Längengrad</value>
   </data>
   <data name="PropertyLatitudeRef" xml:space="preserve">
-    <value>Latitude Ref</value>
+    <value>Breitengrad</value>
   </data>
   <data name="PropertyLongitudeRef" xml:space="preserve">
-    <value>Longitude Ref</value>
+    <value>Längengrad</value>
```
Comment (chingucoding) [inline src/Files.Uwp/Strings/de-DE/Resources.resw:913]: Uhm, why is "Ref" included in the english value?

**Review by chingucoding — APPROVED** [review]

**Pull request merged**